### DISCUSSION TITLE
I generated a Development documentation wiki with AI for baby buddy

### DISCUSSION THREAD

**Discussion by barakplasma** [babybuddy/babybuddy#773] [Show and tell]
I wanted to play with mutable ai, so I had it generate a wiki for baby buddy.

The Development documentation wiki it generated helps explain the project file structure and what select files do. It would help new developers become familiar with the codebase

Wiki available at https://wiki.mutable.ai/babybuddy/babybuddy

**Comment by cdubz** [comment]
Interesting! That is extremely detailed and actually provides some useful context for things I forget about hah. How does this work? Is it just generated once like this or updated somehow? Seems worth adding to [BBotW](https://github.com/babybuddy/babybuddy?tab=readme-ov-file#baby-buddy-on-the-web).

**Reply by barakplasma** [reply]
I don't think it's auto-updated, but I might be wrong. I will reach out to the start up and find out.
I made a [PR](https://github.com/babybuddy/babybuddy/pull/776) to update the BBotW section of the README.
